docs/UserGuide.md

```diff
@@ -44,13 +45,34 @@ It can also keep track of what ingredients you have on hand, and recommend recip
 
 ## Features
 
-### Auto-Save Databases
+### Recipes and Ingredients
+* There are 2 categories of data that YMFC will help you store: Recipes and Ingredients
+* The **Recipes** are stored in the **cookbook** database while **Ingredients** are stored in the **inventory** database
```

Comment:
Here you mentioned about "cookbook" and "inventory". Should we rename our savefile that way for easier referencing?